### PR TITLE
Use mainline wasmd

### PR DESCRIPTION
**Pull request by the-frey**
WIP on this as it's late here.

Creating a PR so I can use the docker builds for further testing.

Builds and passes CI but needs further testing.

### CODE REVIEW

**Comment by the-frey** [comment]
I removed the ordered endblockers because I wasn't clear why they needed to be spelled out - can revert if that is a mistake.

**Review by faddat — APPROVED** [review]
Great!  How should we test this next?

**Comment by the-frey** [comment]
I'm going to run some local tests and deploy some smart contracts. I would guess that if others are happy we move to upgrade uni next week?

File: app/app.go [hunk 1]
```diff
@@ -488,11 +488,29 @@ func New(
...
+		paramstypes.ModuleName,
+		vestingtypes.ModuleName,
+		// additional modules
+		ibchost.ModuleName,
+		ibctransfertypes.ModuleName,
+		wasm.ModuleName,
 	)
 
-	app.mm.SetOrderEndBlockers(crisistypes.ModuleName, govtypes.ModuleName, stakingtypes.ModuleName)
+	// app.mm.SetOrderEndBlockers(crisistypes.ModuleName, govtypes.ModuleName, stakingtypes.ModuleName)
```
[the remark this answers is not in the document]
Comment (the-frey) [reply app/app.go]: ahhh that does make sense. That was the only bit I thought was pretty fishy... makes sense that it was supposed to be there.

**Pull request merged**